### ISSUE TITLE
vmm-sys-utils dependency problem

### ISSUE THREAD

**Issue by studychao** [rust-vmm/kvm-bindings#74]
The vmm-sys-utils used by kvm-bindings is  `vmm-sys-util = { version = ">=0.8.0", optional = true }` and yesterday vmm-sys-utils upgraded to v0.11.0 from v0.10.0 which leads the vmm-sys-util used by kvm-binding upgraded to v0.11.0 without user notice.

And Dragonball VMM use kvm-bindings with other crates which specify the vmm-sys-util version to 0.10.0 so that yesterday's upgrade of vmm-sys-utils caused the compile error of Dragonball.

Could we also specify the version for `vmm-sys-utils` in `kvm-bindings`(i.e. use  caret dependencies )?

**Comment by andreeaflorescu** [comment]
Yes, we should fix the way dependencies are specified here as well. Do you mind sending a PR? I can review it, and we can publish a new version.
